srcts/src/shiny/bind.ts

```diff
@@ -204,10 +223,17 @@ function bindInputs(
       if (el.hasAttribute("data-shiny-no-bind-input")) continue;
       const id = binding.getId(el);
 
-      // Check if ID is falsy, skip
+      // Don't bind if ID is falsy...
       if (!id) continue;
 
-      bindingsRegistery.addBinding(id, "input");
```

Comment:
The change here would cause currently-bound inputs to both rebind and to increment the binding registry count, unless an intervening `unbindAll()` happened in between.

Which ultimately would conflate binding the same binding to the same element (it's already bound so the binding should no-op) with binding a _new element_ with the same ID.